podTemplates/lib-test-pod.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: lib-test-pod
+  annotations:
+    cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
+  labels:
+    extra-cpu: "true"
+
+spec:
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: extra-cpu
+                operator: In
+                values:
+                  - true
+          topologyKey: kubernetes.io/hostname
+
+  containers:
+    - name: java-build
+      image: maven:3.8.6-openjdk-11-slim
+      imagePullPolicy: Always
+      command:
+        - cat
+      tty: true
+      resources:
+        requests:
```

Comment:
Sinc you're requesting extra cpu, would it make sense to add more resources here too?

Reply:
it was a copy-paste, I don't think this has to have that rule.